src/commands/ssh.ts

```diff
@@ -163,11 +163,12 @@ const ssh = async (args: yargs.ArgumentsCamelCase<SshCommandArgs>) => {
   // Prefix is required because the backend uses it to determine that this is an AWS request
   const authn = await authenticate();
   await validateSshInstall(authn);
-  const response = await request(
+  const response = await request<AwsSsh>(
     {
       ...pick(args, "$0", "_"),
       arguments: [
         "ssh",
+        "session",
```

Comment:
The signature of the ssh yargs command changed from `ssh <destination>` to `ssh session <destination>`.